pkg/codegen/dotnet/utilities.go

```diff
@@ -78,3 +80,45 @@ func makeValidIdentifier(name string) string {
 func propertyName(name string) string {
 	return makeValidIdentifier(Title(name))
 }
+
+func makeSafeEnumName(name string) (string, error) {
+	safeName := name
+
+	// If the name is one illegal character, replace it.
+	if len(safeName) == 1 && !isLegalIdentifierStart(rune(safeName[0])) {
+		enumReplacer := strings.NewReplacer(
```

Comment:
Nit: This could also be implemented using a `map[string]string` (or even `map[rune]string`) lookup, but what you have is fine with me.